From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andy Locascio <andy@slack-corp.com>
Date: Thu, 9 Jan 2020 15:48:36 -0800
Subject: remove deprecated task API override removed in latest v8

Node.js will pick this up when they roll next.

CL: https://chromium-review.googlesource.com/c/v8/v8/+/1868620

diff --git a/src/node_platform.h b/src/node_platform.h
index 24f7b337bb8fd7825cda4a4bfc9bafeb0f7aece2..599de93f69d2ef73b833c9138c9640202ef167a7 100644
--- a/src/node_platform.h
+++ b/src/node_platform.h
@@ -147,14 +147,6 @@ class NodePlatform : public MultiIsolatePlatform {
   void CallOnWorkerThread(std::unique_ptr<v8::Task> task) override;
   void CallDelayedOnWorkerThread(std::unique_ptr<v8::Task> task,
                                  double delay_in_seconds) override;
-  void CallOnForegroundThread(v8::Isolate* isolate, v8::Task* task) override {
-    UNREACHABLE();
-  }
-  void CallDelayedOnForegroundThread(v8::Isolate* isolate,
-                                     v8::Task* task,
-                                     double delay_in_seconds) override {
-    UNREACHABLE();
-  }
   bool IdleTasksEnabled(v8::Isolate* isolate) override;
   double MonotonicallyIncreasingTime() override;
   double CurrentClockTimeMillis() override;
